packages/core/upload/admin/src/hooks/useConfig.js

```diff
@@ -0,0 +1,55 @@
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { useNotification, useTracking } from '@strapi/helper-plugin';
+
+import { axiosInstance } from '../utils';
+import pluginId from '../pluginId';
+
+const endpoint = `/${pluginId}/configuration`;
+const queryKey = [pluginId, 'configuration'];
+
+export const useConfig = () => {
+  const queryClient = useQueryClient();
+  const { trackUsage } = useTracking();
+  const toggleNotification = useNotification();
+
+  const config = useQuery(
+    queryKey,
+    async () => {
+      const res = await axiosInstance({
+        method: 'get',
+        url: endpoint,
+      });
+
+      return res.data;
+    },
+    {
+      onError: () =>
+        toggleNotification({
+          type: 'warning',
+          message: { id: 'notification.error' },
+        }),
+    }
+  );
+
+  const putMutation = useMutation(
+    (body) => {
+      axiosInstance.put(endpoint, body);
+    },
+    {
+      onSuccess() {
```

Comment:
NIT, lets make this consistent by using an arrow function? 😄

Reply:
I agree but eslint wants me to keep this as `onSuccess() {}`. I've changed all the other functions in this hook to use the same syntax